Sources/Host/Darwin/LibProc.cpp

```diff
@@ -55,7 +58,23 @@ void LibProc::EnumerateProcesses(
 }
 
 std::string LibProc::GetThreadName(ProcessId pid, ThreadId tid) {
-  return "<unknown>";
+  std::string dft("<unknown-thread>");
```

Comment:
Can you use `<unknown>` instead of `<unknown-thread>`? I just changed our default strings to be more uniform.